yt_dlp/extractor/twitter.py

```diff
@@ -448,6 +498,23 @@ class TwitterIE(TwitterBaseIE):
             'view_count': int,
         },
         'add_ie': ['TwitterBroadcast'],
+    }, {
+        # Twitter Spaces
+        'url': 'https://twitter.com/MoniqueCamarra/status/1550101959377551360',
+        'info_dict': {
+            'id': '1lPJqmBeeNAJb',
+            'ext': 'm4a',
+            'title': 'EuroFile@6 Ukraine Up-date-Draghi Defenestration-the West',
+            'uploader': r're:Monique Camarra.+?',
+            'uploader_id': 'MoniqueCamarra',
+            'live_status': 'was_live',
+            'description': 'md5:c62fc4c35ce2e0e977d5a72fc3418594',
+            'timestamp': 1658407771464,
+        },
+        'add_ie': ['TwitterSpaces'],
+        'params': {
+            'skip_download': True,  # requires ffmpeg
+        },
```

Comment:
I would maybe add the test to the back, as to not change test numbering? For example I've referenced 18 and 19 before, but they moved up to 19 and 20 now.